### PR TITLE
[MDS-4599] View all documents for a project.

### PR DESCRIPTION
**Pull request by areyeslo**
## Objective 

[MDS-4599](https://bcmines.atlassian.net/browse/MDS-4599)

Documents Tab to show the list of all documents related to a Project (Project Description, IRT, Major Mine Application)
image

### CODE REVIEW

File: services/minespace-web/src/components/pages/Project/DocumentsTab.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    const { projectGuid } = this.props.match?.params;
+    return this.props.fetchProjectById(projectGuid);
+  };
+
+  handleTabChange = (activeTab) => {
+    const { projectGuid } = this.props.match.params;
+    const url = `/projects/${projectGuid}/documents/#${activeTab}`;
+    this.props.history.push({ pathname: url });
+
+    const a = document.createElement("a");
```
Comment (BenHarrington) [inline services/minespace-web/src/components/pages/Project/DocumentsTab.js:52]: Why are we creating an anchor and clicking it here? My understanding was that the history.push() should suffice.

**Comment by BenHarrington** [comment]
Looking really good. 

I noticed we are missing unit tests for the new components DocumentsPage & DocumentsTab, but I don't see much else to change here, excellent work. 

Big thanks for adding in all the missing stuff for DocumentViewer.

Did you perform a regression at all by any other features that would be effected by the DocumentsTable change(seen that Variances spec for example was changed), or are you confident everything should behave the same?

**Comment by areyeslo** [comment]
Tested the following components using DocumentTable component in minespace-web:
- Variance Details
- Major Mine Application Review & Submit
- Project Summary: Document Upload
- View File History modal for IRT
- IRT File Import
- View All Documents (Project Description, IRT, MMA)
- Project Overview

**Review by BenHarrington — APPROVED** [review]
Code looks great, and if you are happy with the backwards compatibility then I am good to approve this. Awesome job Arturo!

**Pull request merged**